src/platform/forms-system/src/js/helpers.js

```diff
@@ -82,13 +82,19 @@ export function getInactivePages(pages, data) {
 }
 
 export function createFormPageList(formConfig) {
-  return Object.keys(formConfig.chapters).reduce((pageList, chapter) => {
-    const chapterTitle = formConfig.chapters[chapter].title;
-    const pages = Object.keys(formConfig.chapters[chapter].pages).map(page => ({
-      ...formConfig.chapters[chapter].pages[page],
+  if (!formConfig?.chapters) return [];
+
+  return Object.keys(formConfig.chapters).reduce((pageList, chapterKey) => {
```

Comment:
Thanks for renaming this to `chapterKey`. Seems more correct than calling it chapter.